src/index.html

```diff
@@ -11,7 +11,163 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-group">
+        <legend>Personal information</legend>
+        <div class="form-fild">
+          <label for="Surname">Surname:</label>
+          <input
+            type="text"
+            name="Surname"
+            autocomplete="off" 
+            id="Surname"
+          >
+        </div>
+        <div class="form-fild">
+          <label for="Name">Name:</label>
+          <input 
+            type="text" 
+            name="Name" 
+            autocomplete="off" 
+            id="Name"
+          >
+        </div>
+        <div class="form-fild">
+          <label for="How old are You?">How old are You?</label>
+          <input
+            type="number" 
+            name="How old are You?" 
+            min="1" 
+            max="100" 
+            value="12" 
+            id="How old are You?"
+          >
+        </div>
```

Comment:
there is no need to wrap each field into div, you can just put most of inputs inside the label